packages/web/src/tabpanel/TabsPanel.svelte

```diff
@@ -224,7 +224,13 @@
     category: 'Tabs',
     name: 'Close tab',
     keyText: isElectronAvailable() ? 'CtrlOrCommand+W' : null,
-    testEnabled: () => getOpenedTabs().filter(x => !x.closedTime).length >= 1,
+    testEnabled: () => {
+      const hasAnyOtherTab = getOpenedTabs().filter(x => !x.closedTime).length >= 1;
+      const hasConfirmModalOpen = getOpenedModals().some(x => x.component === CloseTabModal);
```

Comment:
const hasModalOpen = getOpenedModals().length > 0;

in fact, you cannot close tab, if any modal is open